spec/models/skill_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+RSpec.describe Skill, type: :model do
+  let(:role) { FactoryGirl.create(:role) }
+  let(:user) { FactoryGirl.create(:user, role_id: role.id) }
+  let(:skill) { FactoryGirl.create(:skill, user_id: user.id) }
+
+  it 'invalid if power value is string' do
```

Comment:
Read about Rspec shared examples. I guess they would fit there instead of duplication for each parameter